package.json

```diff
@@ -15,29 +15,41 @@
   "dependencies": {
```

Comment:
There are things that can be moved to devDependencies, like he babel-parser, preset, etc. jest stuff, webpack stuff (like style-loader)